docs/user-guide/configure-instance-directory.md

```diff
@@ -160,6 +163,15 @@ Refer to detailed section about [API Gateway configuration](api-mediation/api-ga
 
 - `EXTERNAL_COMPONENTS`: For third-party extenders to add the full path to the directory that contains their component lifecycle scripts.  For more information, see [Zowe lifecycle - Zowe extensions](../extend/lifecycling-with-zwesvstc.md#zowe-extensions).
 
+### High Availability
+
+The high availability (HA) feature of Zowe is under development and has not been fully delivered.  The following values are work in progress towards HA capability, they are not used and will be documented in more detail once more HA support is finalized in a future Zowe release. 
+
+- `ZWE_DISCOVERY_SERVICES_LIST` : 
```

Comment:
`ZWE_DISCOVERY_SERVICES_LIST` is used, so needs to be left untouched. I'm concerned if they read this and don't understand they might delete the properties, which would make Zowe break

Reply:
Added a warning to let sysprogs know not to touch it